recipe/meta.yaml

```diff
@@ -18,17 +18,19 @@ build:
 requirements:
   host:
     - pip
-    - python
-    - setuptools >=40.6.0
+    - python >=3.7
```

Comment:
Since this is _not_ a `noarch: python` package (see the `build` block above), we should drop the `>=3.7` qualifier from `python` in the `host` and `run` requirements. (I.e., the previous, just plain `python` requirement is correct.)

The Python version constraints will automatically be imposed by `conda build` (via cbc.yaml) when the package is built.

Reply:
Ahh I see. That makes sense. Thanks, I'll go ahead and make this amendment.